translator/config/schema.go

```diff
@@ -983,6 +983,10 @@ func GetJsonSchema() string {
 	return schema
 }
 
+func OverwriteSchema(newSchema string) {
```

Comment:
I do not see where calls this method and there is no description how to use it.

Could you add some description about how to use it?

Reply:
```
import (
	parent "github.com/aws/amazon-cloudwatch-agent/translator/config"
)

func applyMyPluginSchema(oldSchema string) string {
    // Update oldSchema with new changes
}

func init() {
	//We interfere external schema by adding updated schema changes
	parent.OverwriteSchema(applyMyPluginSchema(parent.GetJsonSchema()))
}
```
We are working on a new input plugin which needed some changes on existing schema.
The above **OverwriteSchema** method allow us to add our schema changes to existing one.

Reply:
Is your plugin name fixed?

If yes, shall we just enhance the existing schema?

Reply:
Confirmed with arlnav, they have a private plugin, it is fine.

Reply:
OverwriteSchema will allow anyone to override existing schema with their own schema when they roll out their own version of plugin instead of hardcoding existing schema, specifically when new schema changes are not for public use.